tests/integration/targets/homebrew/tasks/formulae.yml

```
@@ -363,3 +363,27 @@
         - "reinstall_result.msg == 'Package already installed: sqlite3'"
         - "reinstall_result.changed_pkgs == []"
         - "reinstall_result.unchanged_pkgs == ['sqlite3']"
+
+# Test with homebrew tap
+- block:
+  - name: Tap terraform homebrew repository
+    community.general.homebrew_tap:
+      name: hashicorp/tap
+    become: true
+    become_user: "{{ brew_stat.stat.pw_name }}"
+
+  - name: Install homebrew tap
+    community.general.homebrew:
+      name: hashicorp/tap/terraform
+      state: latest
+      update_homebrew: false
+    register: terraform_install_result
+    become: true
+    become_user: "{{ brew_stat.stat.pw_name }}"
+
+  - assert:
+      that:
+        - terraform_install_result is changed
+        - "terraform_install_result.msg == 'Package upgraded: hashicorp/tap/terraform'"
+        - "terraform_install_result.changed_pkgs == ['hashicorp/tap/terraform']"
+        - "terraform_install_result.unchanged_pkgs == ['']"
```

Shouldn't this be
```suggestion
        - "terraform_install_result.unchanged_pkgs == []"
```
?

---

Absolutely, the error is kinda masked because the job did a retry and so the second time the error is different.

```logs
> 04:40 TASK [homebrew : Install homebrew tap] *****************************************
04:49 changed: [testhost] => {"changed": true, "changed_pkgs": ["hashicorp/tap/terraform"], "msg": "Package upgraded: hashicorp/tap/terraform", "unchanged_pkgs": []}
04:49 
04:49 TASK [homebrew : assert] *******************************************************
04:49 fatal: [testhost]: FAILED! => {
04:49     "assertion": "terraform_install_result.unchanged_pkgs == ['']",
04:49     "changed": false,
04:49     "evaluated_to": false,
04:49     "msg": "Assertion failed"
04:49 }
04:49 

 WARNING: Retrying test target "homebrew" with maximum verbosity.
```